test/dummy/db/schema.rb

```diff
@@ -11,22 +11,22 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20130715002029) do
```

Comment:
This happened because I actually ran the migrations in the dummy app... which probably wasn't necessary.